### PR TITLE
Zombie process kill

### PR DESCRIPTION
**Pull request by RomanLevkovych**
When process finishes, its' info still in the system and we have to clear it.

### CODE REVIEW

File: applications/ExecutionManager/include/os_interface.hpp [hunk 1]
```diff
@@ -13,8 +13,7 @@ class OsInterface : public IOsInterface
   pid_t fork() override;
   int execv(const char* path, char* argv[]) override;
   int kill(pid_t procId, int signal) override;
-
-  ~OsInterface() override = default;
+    ~OsInterface() override = default;
```
Comment (ihor96berizka) [inline applications/ExecutionManager/include/os_interface.hpp]: looks like here is redundant white space

File: applications/ExecutionManager/src/em_main.cpp [hunk 2]
```diff
@@ -4,9 +4,15 @@
 #include "os_interface.hpp"
 #include <iostream>
 #include <memory>
+#include <csignal>
```
Comment (ihor96berizka) [inline applications/ExecutionManager/src/em_main.cpp]: for consistency: check if somewhere else is signal.h and replace with csignal.

File: applications/ExecutionManager/src/em_main.cpp [hunk 3]
```diff
@@ -4,9 +4,15 @@
...
+#include <logger.hpp>
+
 
 int main(int argc, char **argv)
 {
+  if (signal(SIGCHLD, SIG_IGN) == SIG_ERR) {
```
Comment (ihor96berizka) [inline applications/ExecutionManager/src/em_main.cpp]: ::signal(...)

File: applications/MachineStateManager/src/msm_main.cpp [hunk 4]
```diff
@@ -3,7 +3,8 @@
 #include <logger.hpp>
 #include <constants.hpp>
 
-#include <signal.h>
+#include <csignal>
+#include <iostream>
 #include <chrono>
```
Comment (rBekesh) [inline applications/MachineStateManager/src/msm_main.cpp]: not needed

File: applications/MachineStateManager/src/msm_main.cpp [hunk 5]
```diff
@@ -3,7 +3,8 @@
 #include <logger.hpp>
 #include <constants.hpp>
 
-#include <signal.h>
+#include <csignal>
+#include <iostream>
```
Comment (rBekesh) [inline applications/MachineStateManager/src/msm_main.cpp]: not needed

**Pull request merged**